File: Python/String/CountVowels.py
#######################################################################################################################
#
# Accept the String from user and count number of vowels.
#
#######################################################################################################################




def Count(str):
    iCount=0
    for i in str:
        if((i=='A')or(i=='E')or(i=='I')or(i=='O')or(i=='U')or(i=='a')or(i=='e')or(i=='i')or(i=='o')or(i=='u')):
            iCount=iCount+1

    return iCount


if __name__=="__main__":
    iRet=0
    str=input("Enter the String\n")
    iRet=Count(str)
    print("Number of Vowels are:",iRet)

File: Java/String/CountVowels.java
/////////////////////////////////////////////////////////////////////////////
//
// Accept the String from user and count number of vowels. 
//
////////////////////////////////////////////////////////////////////////////


import UserPackage.StringX;
class Count extends StringX
{
    public int CountVowels()
    {
        int iCnt=0,i=0;
       char Arr[]=str.toCharArray();
       for(i=0;i<Arr.length;i++)
       {
         if((Arr[i]=='A')||(Arr[i]=='E')||(Arr[i]=='I')||(Arr[i]=='O')||(Arr[i]=='U')||(Arr[i]=='a')||(Arr[i]=='e')||(Arr[i]=='i')||(Arr[i]=='o')||(Arr[i]=='u'))
         {
            iCnt++;
         }
           
       }
       return iCnt;
    }

}
class CountVowels
{
    public static void main(String arg[])
    {
        int iRet=0;
       Count cobj=new Count();
       cobj.Accept();
       iRet=cobj.CountVowels();
       System.out.println("The number of vowels are:"+iRet);

    }
}